rootfs/opt/kubectl/example/GoogleBackup/configmap/backup.sh

```diff
@@ -0,0 +1,19 @@
+set -e
+set -x
+
+echo "Start backup"
+
+gcloud auth activate-service-account --key-file=/secret-backup/neural-theory-215601-53bf50004612.json
+
+MYSQL_SOCK="mysql/mysql.sock"
+gsutil -m rsync -P -x "$MYSQL_SOCK" -r /qserv/data gs://qserv-backup/qserv-dev/"$HOSTNAME"
+
+STATUS_FILE="/tmp/$HOSTNAME-backup-ok"
+touch "$STATUS_FILE"
+gsutil -m cp -r "$STATUS_FILE" gs://qserv-backup/qserv-dev/results/
+
+echo "Succeed backup"
+while true
```

Comment:
The goal for backup is to mimic Qserv app so so that backup pods attach automatically to the same PVC than Qserv pods, and restore pods automatillacy create PVC that we be used by Qserv. It is kind of hacky but avoid to explicitly link each backup/restore pod to a PVC by using the same PVCtemplates for backup/restore app and Qserv. Am I clear enough? If not we can talk about it later.